packages/vike-react/renderer/types.d.ts

```diff
@@ -20,12 +20,15 @@ declare global {
       /** &lt;title>${title}&lt;/title> - set by onBeforeRender() hook, has precedence over the config */
       title?: string
 
+      lang?: string
+
       userAgent?: string
 
-      // Needed by getTitle()
+      // Needed by getTitle() & getLang()
       data?: {
         /** &lt;title>${title}&lt;/title> - set by data() hook, has precedence over the onBeforeRender() hook */
         title?: string
+        lang?: string
```

Comment:
As said in other comment, I think we don't need to support setting `lang` in a `data()` hook?

Reply:
Removed 👍